easybuild/easyconfigs/n/Nextflow-pipeline/Nextflow-pipeline-20221110-foss-2021a-mag.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'Bundle'
+
+name = 'Nextflow-pipeline'
```

Comment:
@smoretti How about using `nf-core-mag` as software name (and dropping the `-mag` versionsuffix)?

Not sure which approach is better...

@ocaisa?

Reply:
To me Nextflow-pipeline is not defined enough, there are going to be a gazillion of those so all the distinction would have to be in a versionsuffix. Best to use what they use, just a pity about the /

Reply:
right, make sense